conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
 - libclang=8.0.0
 - cmake=3.14.5
 - numba=0.45*
-- rapidsai/label/cuda10.0::cupy>=6*
+- cupy>=6.5
```

Comment:
should we make this `>=6.5,<7.0`? I'm not sure we're actually ready for 7.0 if it pops out one day soon